### PR TITLE
revise readme

### PR DESCRIPTION
**Pull request by rambohe-ch**
/kind documentation

#### What this PR does / why we need it:
typo fix

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The above figure demonstrates the core OpenYurt architecture. The major componen
   `YurtAppDaemon` enables to deploy workload for every matching NodePool.
   `YurtIngress` orchestrates the deployment of multiple ingress controllers to the respective NodePools.
 - **[Raven/Raven Controller Manager](https://openyurt.io/docs/next/core-concepts/raven)**: It is focused on edge-edge and edge-cloud communication in OpenYurt. and provide layer 3 network connectivity among pods in different physical regions, as there are in one vanilla Kubernetes cluster.
-- **Pool-Coordinator**: On instance of Pool-Coordinator is deployed in every edge NodePool, and in conjunction with YurtHub to provide heartbeat proxy, cloud edge traffic reuse abilities, etc.
+- **Pool-Coordinator**: One instance of Pool-Coordinator is deployed in every edge NodePool, and in conjunction with YurtHub to provide heartbeat delegation, cloud edge traffic reuse abilities, etc.
```
Comment (Congrool) [inline README.md:59]: We originally call `cloud edge traffic reuse` as `cloud-edge traffic multiplexing` in proposal.

Comment (Congrool) [reply README.md:59]: It seems to use `cloud-edge traffic multiplex` will be better, since multiplex can be noun.

Comment (rambohe-ch) [reply README.md:59]: fixed

**Pull request merged**